vm_setup_vars.yml

```diff
@@ -51,6 +51,8 @@ provisioning_network:
     bridge: "{{ provisioning_network_name  }}"
     forward_mode: bridge
 
+virsh_dhcp_enabled: "{{ lookup('env', 'DHCP_ENABLED')|default('', true) }}"
```

Comment:
I don't think we need this, since the environment variable is already respected in the defaults ref https://github.com/metal3-io/metal3-dev-env/pull/879 ?